src/inputs/input.css

```diff
@@ -0,0 +1,174 @@
+}
+
+input.omrs-input-underlined {
+  background: var(--omrs-color-ink-white);
+}
+
+input.omrs-input-underlined ~ label,
+input.omrs-input-filled ~ label,
+input.omrs-input-outlined ~ label {
+  color: var(--omrs-color-ink-medium-contrast);
+  font-size: 1.0625rem;
+  font-weight: normal;
+  position: absolute;
+  pointer-events: none;
+  top: 1.25rem;
+  transition: 0.2s ease all;
+  padding-left: 0.875rem;
+  letter-spacing: 0.44px;
+  color: var(--omrs-color-ink-medium-contrast);
+}
+
+input.omrs-input-underlined:focus ~ label,
+input.omrs-input-filled:focus ~ label,
+input.omrs-input-underlined:valid ~ label,
+input.omrs-input-filled:valid ~ label {
+  top: 0;
+  font-size: 0.9375rem;
+}
+
+input.omrs-input-outlined:focus ~ label,
+input.omrs-input-outlined:valid ~ label {
+  top: -0.75rem;
+  left: 0.875rem;
+  font-size: 0.9375rem;
+
+  /** for now, easiest way to show label ontop of border **/
+  background: white;
+  padding: 0.25rem;
+}
+
+/* input.omrs-input-outlined:valid ~ label {
+	display: none;
+} */
+
+input.omrs-input-underlined:hover:not(:focus):not(:disabled),
+input.omrs-input-filled:hover:not(:focus):not(:disabled) {
+  background-color: var(--omrs-color-interaction-minus-two);
+  border-bottom-color: var(--omrs-color-ink-high-contrast);
+}
+
+input.omrs-input-outlined:hover:not(:focus):not(:disabled) {
+  border-color: var(--omrs-color-ink-medium-contrast);
+}
+
+input.omrs-input-underlined:disabled,
+input.omrs-input-filled:disabled,
+input.omrs-input-outlined:disabled {
+  cursor: not-allowed;
+}
+
+input.omrs-input-underlined:disabled,
+input.omrs-input-filled:disabled {
+  background-color: var(--omrs-color-bg-low-contrast);
+  border-color: var(--omrs-color-ink-low-contrast);
+}
+
+input.omrs-input-outlined:disabled {
+  border-color: var(--omrs-color-ink-lowest-contrast);
+}
+
+input.omrs-input-underlined:disabled ~ label,
+input.omrs-input-filled:disabled ~ label,
+input.omrs-input-outlined:disabled ~ label,
+input.omrs-input-underlined:disabled ~ span.omrs-input-helper,
+input.omrs-input-outlined:disabled ~ span.omrs-input-helper,
+input.omrs-input-filled:disabled ~ span.omrs-input-helper {
+  color: var(--omrs-color-ink-low-contrast);
+}
+
+input.omrs-input-danger {
+  border-color: var(--omrs-color-danger);
+}
+
+input.omrs-input-danger ~ label,
+input.omrs-input-danger ~ span.omrs-input-helper {
+  color: var(--omrs-color-danger);
+}
+
+input.omrs-input-danger svg {
+  fill: var(--omrs-color-danger);
+}
+
+input.omrs-input-underlined:disabled svg,
+input.omrs-input-filled:disabled svg,
+input.omrs-input-outlined:disabled svg {
+  fill: var(--omrs-color-ink-low-contrast);
+}
+
+input.omrs-input-underlined svg,
+input.omrs-input-filled svg,
+input.omrs-input-outlined svg {
+  position: absolute;
+  vertical-align: middle;
+  top: 1rem;
+  right: 0.875rem;
+  fill: grey;
+}
+
+input.omrs-input-underlined:focus:not(.omrs-input-danger) svg,
+input.omrs-input-filled:focus:not(.omrs-input-danger) svg {
+  fill: black;
```

Comment:
Looking at Figma, I think this maybe should be `var(--omrs-color-ink-high-contrast)`??